### PR TITLE
Add implementation using webcomponent

### PR DESCRIPTION
**Pull request by tenosiswono**
Implementation Hacker News PWA using Webcomponentjs and Redux

Github Repo: https://github.com/tenosiswono/hnpwa-wc
Running Application: https://hnpwa-wc.firebaseapp.com
lighthouse: [91/100](https://www.webpagetest.org/lighthouse.php?test=180123_ZC_f6d860c8a63a42ff405aa32ad38d99c6&run=2)
interactive-em: [3,3s](https://www.webpagetest.org/result/180123_HE_8ec4ffc329ea532f9101e469c9fe6c20/)
interactive-faster-3g: [2,4s](https://www.webpagetest.org/result/180123_HE_8ec4ffc329ea532f9101e469c9fe6c20/)

I'm trying to implement Webcomponent es6 classes with Redux and bundling with Webpack.
Maybe my code is messy. Please let me know if anything needs to be done or not.
Thanks!

### CODE REVIEW

**Comment by usm4n** [comment]
Hi @tenosiswono, I had a chance to go through your submission, the first thing that I noted is that the application won't load in firefox. It is a requirement that app must do its best to work cross browser. You will need a polyfil for that.

Here is the screen shot of what I am currently seeing on Firefox 57:
firefox_scrot

Please check these projects for help:
https://github.com/webcomponents/webcomponentsjs
https://github.com/dsolimando/hnpwa-mobileelements/tree/pagination

Another strange bug that I have found on chrome is that selecting the Top section from inside the comments page of a perticular post renders the Next link as disabled as seen below:
firefox_scrot2

Another feature that would be nice is to have collapsible comments, though it is not a strict requirement.

**Comment by tenosiswono** [comment]
Hi @usm4n Thanks
My bad. I didn't check in firefox
I already make a fix for all of those bugs.

File: site/_apps/hnpwa-wc.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+service-worker: 'Application Shell with Workbox'
+perfomance-patterns: 'Server push, preload'
+server-side-rendering: 'No'
+api: 'hnpwa.com/api'
+hosting: 'Firebase'
+authors: 
+    - name: 'Teno'
+lighthouse: '91/100'
+interactive-em: '3.1s'
+interactive-faster-3g: '2.4s'
```
Comment (housseindjirdeh) [inline site/_apps/hnpwa-wc.txt]: With newer traces these numbers seem to be a bit slower (especially EM/Slow 3G). Could you please update the numbers here? 

Regular 3G: https://www.webpagetest.org/result/180629_2S_24c6bb9620c25086dd87346c1f759428/
Slow 3G: https://www.webpagetest.org/result/180629_7A_11ffcb6fe41678ac561bd9daae06f8a1/

**Comment by housseindjirdeh** [comment]
Since we're in the process of clearing our issues/PRs @tenosiswono, I'll close this for now. If you happen to be interested in landing this submission, please let me know and I'll reopen it!

**Pull request closed without merging**